src/api/quoter/quote/quote.ts

```diff
@@ -122,7 +130,7 @@ export class Quote {
     }
 
     getPreset(type = PresetEnum.fast): Preset {
-        return this.presets[type]
+        return this.presets[type] as Preset
```

Comment:
📓 typescript hack required to avoid problems with

```
 [PresetEnum.custom]: response.presets.custom
                ? new Preset(response.presets.custom)
                : undefined

```